src/main/java/uk/gov/justice/digital/delius/service/CaseNoteService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.delius.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class CaseNoteService {
+
+    private final RestTemplate restTemplate;
+
+    @Autowired
+    public CaseNoteService(@Qualifier("alfrescoRestTemplateWithAuth") RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public void upsertCaseNotesToDelius(final String nomisId, final Long caseNotesId, final String caseNote) {
```

Comment:
Couldn't we model the object rather than a JSON string?

Reply:
is there any point?  We're just acting as a proxy here so don't really care what the data is...

Reply:
Agree with Peter